syft/execution/translation/threepio.py

```diff
@@ -0,0 +1,56 @@
+from torch import jit
+from pythreepio.threepio import Threepio
+from pythreepio.utils import Command
+from pythreepio.errors import TranslationMissing
+from syft.execution.translation import TranslationTarget
+from syft.execution.translation.abstract import AbstractPlanTranslator
+
+
+class PlanTranslatorThreepio(AbstractPlanTranslator):
```

Comment:
Base class which will allow us to easily add support for other frameworks using threepio later on